src/widgets/sceneviewwidget.cpp

```diff
@@ -394,15 +401,17 @@ void SceneViewWidget::renderScene()
     }
 
     // render fps
-    float fps = 1.0 / dt;
-    float ms = 1000.f / fps;
     spriteBatch->begin();
-    spriteBatch->drawString(font,
-                            QString("%1ms (%2fps)")
-                                .arg(QString::number(ms, 'f', 1))
-                                .arg(QString::number(fps, 'f', 1)),
-                            QVector2D(8, 8),
-                            QColor(255, 255, 255));
+    if (showFps) {
+        float fps = 1.0 / dt;
+        float ms = 1000.f / fps;
+        spriteBatch->drawString(font,
+                                QString("%1ms (%2fps)")
+                                    .arg(QString::number(ms, 'f', 1))
+                                    .arg(QString::number(fps, 'f', 1)),
+                                QVector2D(8, 8),
```

Comment:
Does this size factor even affect anything anymore with all the magic numbers that produces the huge text?